.github/workflows/cd.yaml

```diff
@@ -0,0 +1,58 @@
+name: Deploy Static Website to S3
```

Comment:
How about changing the name to `Deploy Website` and the name of the file to `deploy-website`?

```suggestion
name: Deploy Website
```